src/Symfony/Component/Stopwatch/StopwatchEvent.php

```diff
@@ -236,11 +243,21 @@ private function formatTime(float $time): float
         return round($time, 1);
     }
 
+    /**
+     * Gets the event name.
+     *
+     * @return string The event name
+     */
+    public function getName()
```

Comment:
```suggestion
    public function getName(): string
```

and no need of the `* @return string The event name`

wdyt?

Reply:
Updated that :).
However, I was trying to not be different than the other class methods as anyway the whole class needs these updates.

Reply:
i think for previous/actual API, those typehints are not added only between major version for BC
but as it is a new one, maybe it is ok to add it
(maybe core member will correct me here)